sdk/translation/ai-translation-document-rest/review/ai-translation-document.api.md

```diff
@@ -0,0 +1,640 @@
+## API Report File for "@azure-rest/ai-translation-document"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+/// <reference types="node" />
+
+import { AbortSignalLike } from '@azure/abort-controller';
+import { CancelOnProgress } from '@azure/core-lro';
+import { Client } from '@azure-rest/core-client';
+import { ClientOptions } from '@azure-rest/core-client';
+import { createFile } from '@azure/core-rest-pipeline';
+import { createFileFromStream } from '@azure/core-rest-pipeline';
+import { CreateFileFromStreamOptions } from '@azure/core-rest-pipeline';
+import { CreateFileOptions } from '@azure/core-rest-pipeline';
+import { CreateHttpPollerOptions } from '@azure/core-lro';
+import { ErrorResponse } from '@azure-rest/core-client';
+import { HttpResponse } from '@azure-rest/core-client';
+import { KeyCredential } from '@azure/core-auth';
+import { OperationState } from '@azure/core-lro';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { PathUncheckedResponse } from '@azure-rest/core-client';
+import { RawHttpHeaders } from '@azure/core-rest-pipeline';
+import { RawHttpHeadersInput } from '@azure/core-rest-pipeline';
+import { RequestParameters } from '@azure-rest/core-client';
+import { StreamableMethod } from '@azure-rest/core-client';
+import { TokenCredential } from '@azure/core-auth';
+
+// @public
+export interface BatchRequest {
+    source: SourceInput;
+    storageType?: StorageInputType;
+    targets: Array<TargetInput>;
+}
+
+// @public
+export interface CancelTranslation200Response extends HttpResponse {
+    // (undocumented)
+    body: TranslationStatusOutput;
+    // (undocumented)
+    status: "200";
+}
+
+// @public (undocumented)
```

Comment:
Do you know why none of these are documented? is it generated without docs?

Reply:
I am not sure, this is how it was generated

Reply:
we have an issue to track this in codegen side https://github.com/Azure/autorest.typescript/issues/2610

Reply:
Since there is no action on my side, resolving this comment